index.html

```diff
@@ -7,90 +7,211 @@
+
+
+          <div class="paragraph_description">
+            <p class="text">Every Cabin is up in the nature but also <br> easy to get to and to the towns around</p>
+          </div>
         </div>
       </div>
-    </div>
-    <div class="pros_container">
-      <div class="photo_description_container">
-        <div class="pros_photo"><img class="pros_pic" src="mountains.png" alt="zdjęcie zalet"></div>
-        <div class="paragraph_container">
-          <header class="paragraph_header">Close to nature</header>
-          <p>Every cabin means different areas to explore!</p>
+
+      <div class="pros_container">
+        <div class="photo_description_container">
+          <div class="pros_photo"><img class="pros_pic" src="mountains.png" alt="zdjęcie zalet"></div>
+
+          <div class="header_container">
+            <header class="paragraph_header">Close to nature</header>
+          </div>
+
+          <div class="paragraph_description">
+            <p class="text">Every cabin means different areas to <br> explore!</p>
+          </div>
         </div>
+
       </div>
     </div>
-  </div>
-  <div class="third_page_container">
+  </section>
+
+
+
+
+
+  <section class="third_page_container">
     <div class="left_side">
       <div class="some_text">
-        <header class="third_page_header">Work from somewhere special</header>
-        <p>We gather people who love nature and mountains and help them find perfect spots for a few weeks or even
+        <h3 class="third_page_header">Work from somewhere special</h3>
+        <p class="longer_paragraph">We gather people who love nature and <br> mountains and help them find perfect spots
+          for <br> a few weeks or even
           months.
-          If your job is fully remote and you daydream of holidays, then why sit at home? Work as you normally do and
-          later let yourself dive into nature. Break the habit.</p>
+          If your job is fully <br> remote and you daydream of holidays, then why <br> sit at home? Work as you normally
+          do and
+          later <br> let yourself dive into nature. Break the habit.</p>
+      </div>
+      <div class="btn_third_sector">
+        <button class="book_now" type="submit"><span class="book">Book Now</span></button>
       </div>
-      <div class="btn"><button>Book now</button></div>
     </div>
     <div class="right_side">
       <img class="mountains_pic" src="mountain_home.png" alt="widok na góry">
     </div>
-  </div>
-  <div class="fourth_page_container">
+  </section>
+
+
+
+
+
+  <section class="fourth_page_container">
     <div class="left_side_fourth_page">
       <div><img class="girl_pic" src="girl.png" alt="zdjęcie dziewczyny"></div>
-      <div class="newsletter">
-        <label for="newsletter_header"> newsletter comming soon</label>
-        <input type="email" class="email" placeholder="your e-mail address">
-      </div>
+
     </div>
+    </div>
+
+
+
+
+
     <div class="right_side_fourth_page">
```

Comment:
the page is single, but section is fourth. Avoid naming sections this way. If I request to add another section in-between you'll have to update naming for every other section